tests/test_files/scheduling/task_synchronisation.pfdl

```diff
@@ -20,7 +20,7 @@ Struct CuttingResult
     sheet_parts: SheetPart[]
 End
 
-Struct TransportOrderStep_Struct
+Struct TransportOrderStep
```

Comment:
Using keywords of other plugins will cause problems there, so the Struct needs to have a different name than TransportOrderStep (same for Transport)